EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: eip-xxxx
```

Comment:
```suggestion
eip: 7784
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.